### DISCUSSION TITLE
Swin UNETR intensity window and spacing

### DISCUSSION THREAD

**Discussion by TuanDTr** [Project-MONAI/tutorials#1062] [Q&A] [answered]
I'm looking at the tutorial using Swin UNETR on the BCTV dataset and I find that the intensity window is (-175, 250) and the spacing is (1.5, 1.5, 2). 

(1) Do you use the same setup when you're training on the Medical Segmentation Decathlon (MSD) dataset because I assume you need a special intensity window for each organ?

(2) In case you're using a different setting specific to a task, do you have any rules when selecting these values?

Thanks a lot!

**Comment by tangy5** [comment, marked as answer]
Hi @beantr96 , thanks for the questions.
Yes, it needs to set window for intensities.for each task.
For a rule to select window for each daraset, you could analysis the ground truth mask covered organ/tumor intensities. Calculate a scale then apply a tolerance. 
Thanks
